data/builtins/org.thingpedia.builtin.thingengine.builtin/manifest.tt.in

```diff
@@ -153,6 +161,62 @@ class @org.thingpedia.builtin.thingengine.builtin extends @org.thingpedia.volume
   #[minimal_projection=[]]
   #[confirm=false];
 
+  list query configured_device_info(out id : Entity(tt:device)
```

Comment:
This should be tt:device_id (a unique ID of a device like "com.spotify-gcampax") not tt:device (a Thingpedia device ID like "com.spotify")